src/components/ArticleListing.js

```diff
@@ -15,12 +16,23 @@ class ArticleListing extends Component {
         return (
           <div className={`article-listing`}>
             {hr}
-            <Link to={`${ARTICLE_BASE_URL}${article.slug.current}`}>
+            <Link
+              className={`header-link`}
+              to={`${ARTICLE_BASE_URL}${article.slug.current}`}
+            >
               <h3>
                 {article.title} - By {article.author.name}
               </h3>
-              <p>{article.synopsis}</p>
             </Link>
+            <p>
+              {article.synopsis}{" "}
```

Comment:
Maybe instead of `{" "}` for whitespace:
```suggestion
              {`${article.synopsis} `}
```
or perhaps this:
```suggestion
              {article.synopsis + ' '}
```

Reply:
Sure! I think I like your second example: not as clean, but a little more understandable at a glance.